mod/consensus-types/pkg/types/payload_test.go

```diff
@@ -89,7 +103,15 @@ func TestExecutableDataDeneb_GetTree(t *testing.T) {
 
 func TestExecutableDataDeneb_Getters(t *testing.T) {
 	payload := generateExecutableDataDeneb()
-
+	transactions := make([][]byte, 1)
+	transactions[0] = make([]byte, 0)
+	withdrawals := make([]*engineprimitives.Withdrawal, 1)
+	withdrawals[0] = &engineprimitives.Withdrawal{
```

Comment:
Same for withdrawals[0] , do we need it at this place ?

Reply:
Same with the logic above, here the line that it covers: https://github.com/berachain/beacon-kit/blob/main/mod/consensus-types/pkg/types/payload.ssz.go#L241-L247